include/seqan3/alphabet/adaptation/uint.hpp

```diff
@@ -33,9 +33,8 @@ namespace seqan3::detail
 //!\brief Whether a type is `uint8_t`, `uint16_t` or `uint32_t`.
 //!\ingroup adaptation
 template <typename type>
-constexpr bool is_uint_adaptation_v = std::same_as<type, uint8_t>  ||
-                                      std::same_as<type, uint16_t> ||
-                                      std::same_as<type, uint32_t>;
+constexpr bool is_uint_adaptation_v = std::same_as<type, uint8_t> || std::same_as<type, uint16_t> ||
+    std::same_as<type, uint32_t>;
```

Comment:
this change seems unrelated

Reply:
Resolves: "weird spacing in is_char_adaptation_v, is_uint_adaptation_v"

Reply:
same here